cache/cache.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package cache
+
+import (
+	"sync"
+
+	"github.com/golang/groupcache/lru"
+)
+
+// Cache defines an interface for a cache that stores Measurable content.
+// Eviction only happens when Add() is called, and there's no background
+// goroutine for eviction.
+type Cache interface {
+	// Get tries to find and return data assiciated with key.
+	Get(key string) (data Measurable, ok bool)
```

Comment:
As described [above](https://github.com/keybase/kbfs/pull/1068#discussion_r127488940), it's to avoid panics without having to use `reflect` here. Our key is always a string anyway. But if you feel strongly that this should be an `interface{}`, I can change it into that. Let me know!